packages/rainbowkit/src/wallets/walletConnectors/argentWallet/argentWallet.ts

```diff
@@ -67,22 +67,19 @@ export const argentWallet = ({
           learnMoreUrl: 'https://argent.xyz/learn/what-is-a-crypto-wallet/',
           steps: [
             {
-              description:
-                'Put Argent on your home screen for faster access to your wallet.',
+              description: 'wallet_connectors.qr_code.argent.step1.description',
               step: 'install',
-              title: 'Open the Argent app',
+              title: 'wallet_connectors.qr_code.argent.step1.title',
```

Comment:
For maintenance simplicity, we should flip around the keys here to look something like
`wallet_connectors.argent.qr_code.step1.title`